src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -35,8 +37,8 @@ Item {
 
     property real leftPadding: 16
     property real rightPadding: 16
-    property real topPadding: 12
-    property real bottomPadding: 12
+    property real topPadding: 11
+    property real bottomPadding: 11
```

Comment:
This still seems to position text inside a status input weirdly... (notice how it's not vertically centered anymore). The `bottomPadding` change may not be necessary
<img width="496" alt="Screenshot 2021-09-06 at 09 01 05" src="https://user-images.githubusercontent.com/445106/132174160-24b1b640-0c20-4156-a023-ed20afbe35a9.png">

Reply:
done

Reply:
Okay, for some reason this is still behaving weirdly. Notice how the textfield is "scrollable" even though it shouldn't.

Since `topPadding` and `bottomPadding` have been reverted to what they were before, I believe it has to do something with the introduction of `TextField`. Any idea what could cause this?

Another thing I've noticed: the text of the field itself has quite a big `leftMargin`. What's the reason this was introduced? 

Here's what inputs look like in `master`:
<img width="507" alt="Screenshot 2021-09-06 at 11 16 39" src="https://user-images.githubusercontent.com/445106/132192598-8c1b8e80-898a-42d5-9431-946ce7d1591e.png">

Here's what they look like in this PR:

<img width="500" alt="Screenshot 2021-09-06 at 11 16 57" src="https://user-images.githubusercontent.com/445106/132192694-ef599fae-1996-418e-8de8-84b759a62bc6.png">

Reply:
Scrolling fixed, it seems that the `contentHeight` was not returning the real width and height of text so the flickable was already smaller that the TextField and thus it was flickering a bit, so TextMetrics is used now. Also the TextField sets the cursor x to 10, that's why a left margin of 10 was introduced, however I moved the placeholder out of the TextField and set a 6 left margin it (the difference between 16 and 10) and it should be fine now.